cmd/agent/app/integrations.go

```diff
@@ -404,6 +415,38 @@ func pipCheck(cachePath string) error {
 	return fmt.Errorf("error executing pip check: %v", err)
 }
 
+func moveConfigurationFiles(integration string) error {
+	confFolder := config.Datadog.GetString("confd_path")
+	check := strings.TrimPrefix(integration, "datadog-")
+	confFileDest := filepath.Join(confFolder, fmt.Sprintf("%s.d", check))
```

Comment:
I think this logic to deduce the check name from the wheel name does not work when the check name has underscores `_` in it. Isn't the name of the check defined in the wheel in a way that we can reliably extract from here?

Reply:
Oh you're right, good catch. I'll try to find something

Reply:
`name` field in the `manifest.json` :)